python/lsst/pipe/supertask/activator.py

```diff
@@ -148,7 +148,7 @@ def get_inheritance(stask_class, subclasses=None):
     if isinstance(temp_class, pyclbr.Class):
         subclasses.append(temp_class.name)
         subclasses = get_inheritance(temp_class, subclasses)  # look recursively for super classes
-    else:
+    elif not isinstance(temp_class, pyclbr.Function):
```

Comment:
`cmdLineActivator -lt` crashed for me apparently because for one of the task classes `stask_class.super[0]` returned Function, in which case the old code just added it to the list causing issued later when joining that list. The class was `lsst.pipe.tasks.fakes.BaseFakeSourcesTask`. I think this code needs further improvement to work with cases like that.